install/src/command.rs

```diff
@@ -91,7 +91,10 @@ fn download_to_temp(
     let temp_dir = TempDir::new()?;
     let temp_file = temp_dir.path().join("download");
 
-    let client = reqwest::blocking::Client::new();
+    let client = reqwest::blocking::Client::builder()
+        .connect_timeout(Duration::from_secs(5))
```

Comment:
```suggestion
        .connect_timeout(Duration::from_secs(30))
```

Thanks!  Let's go with 30s.  Someone's bound to try this on WWAN from the sticks